### PR TITLE
LPS-151106 [Raylife AP] Create Status Chart

### PR DESCRIPTION
**Pull request by Mylena-Rodrigues**

### CODE REVIEW

**Comment by Mylena-Rodrigues** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/agent-portal/fragments/application-status/fragment.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+	"configurationPath": "configuration.json",
+	"cssPath": "index.css",
+	"htmlPath": "index.html",
+	"jsPath": "index.js",
+	"name": "Application Status",
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/agent-portal/fragments/application-status/fragment.json:6]: Change the name of directory to applications-status and the property name too.

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/agent-portal/fragments/application-status/index.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+}
+.applications-products-container .applications-products-tooltip {
+	border: 1px solid #55c2ff;
+	color: #55c2ff;
+	font-size: 0.875rem;
+	font-style: normal;
+	width: 100%;
+}
+
+.chart-legend {
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/agent-portal/fragments/application-status/index.css:31]: Please, change it.

From:
```css
.chart-legend {
```

To:
```css
.applications-products-container .chart-legend {
```

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/agent-portal/fragments/application-status/index.js [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+
+	const loadChartData = () => {
+		Promise.allSettled([
+			getApplicationsStatus(CONSTANTS.STATUS.BOUND),
+			getApplicationsStatus(CONSTANTS.STATUS.INCOMPLETE),
+			getApplicationsStatus(CONSTANTS.STATUS.QUOTED),
+			getApplicationsStatus(CONSTANTS.STATUS.REVIEWED),
+			getApplicationsStatus(CONSTANTS.STATUS.UNDERWRITING),
+		]).then((results) => {
+			const getTotalCount = (result) => {
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/agent-portal/fragments/application-status/index.js:46]: Move the `getTotalCount` function out of this function, please.

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/agent-portal/fragments/application-status/index.js [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+
+			const [
+				boundApplicationsResult,
+				incompleteApplicationsResult,
+				quotedApplicationsResult,
+				reviewedApplicationsResult,
+				underwritingApplicationsResult,
+			] = results;
+
+			const colors = {
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/agent-portal/fragments/application-status/index.js:58]: Are there only these statuses for applications?

**Review by silasbsi — CHANGES_REQUESTED** [review]
Looks good to me, but needs some adjustments.

**Pull request closed without merging**